util/formatter.go

```diff
@@ -101,3 +101,16 @@ func FormatEarnings(earnings *iex.Earnings) string {
 
 	return fmtStr
 }
+
+func FormatFuzzySymbols(symbols []iex.SymbolDTO) string {
+	printer := message.NewPrinter(language.English)
```

Comment:
I feel like this is the second time we're making a printer, can you make it so we only initalize this once

Reply:
Why do we even need a printer anyways? why not just use `fmt.Sprintf`

Reply:
printer with english language format gives you some stuff for free like if you give it a number it will add the commas for you "1,000,000"